apps/utils/locust_tests/run-test.sh

```diff
@@ -0,0 +1,30 @@
+export BFD_ENV="3085-test"
+export BFD_ROOT=/home/randall/code/beneficiary-fhir-data
```

Comment:
Should this file be removed/renamed if this was a one-off test?

Reply:
Yes, removed